integration-tests/gradle/src/main/resources/custom-filesystem-provider/application/src/test/java/org/acme/ExampleResourceTest.java

```diff
@@ -1,12 +1,12 @@
 package org.acme;
 
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.is;
 
-import io.quarkus.test.common.QuarkusTestResource;
```

Comment:
Although, I removed the remaining few with misaligned format, having 2 blanks after the package statement

after:
```
package org.acme;

import static io.restassured.RestAssured.given;
```
before:
```
package org.acme;


import static io.restassured.RestAssured.given;
```